presto-iceberg/src/test/java/io/prestosql/plugin/iceberg/TestIcebergSmoke.java

```diff
@@ -105,6 +121,12 @@ public void testDescribeTable()
 
     @Override
     public void testShowCreateTable()
+    {
+        testCreateTableForFormat(ORC);
+    }
+
+    @Test(dataProvider = "allFormats")
+    public void testCreateTableForFormat(FileFormat format)
```

Comment:
It seems this one should be renamed `testShowCreateTable` and method above should be dropped.

Reply:
Unfortunately, as mentioned in the commit comments, the method above is required by `TestIcebergSmoke's` base class `AbstractTestIntegrationSmokeTest`.  `AbstractTestIntegrationSmokeTest` is in module `presto-testing`, and is connector-independent.

`AbstractTestIntegrationSmokeTest` doesn't actually create any tables.  Instead, it uses the `tpch` tables pre-created by the `IcebergQueryRunner`.  We need to figure out a way to run the `AbstractTestIntegrationSmokeTest` methods having pre-created for both `ORC` and `Parquet`.  I'll figure out some way to do it.

Reply:
> Unfortunately, as mentioned in the commit comments, the method above is required by TestIcebergSmoke's base class AbstractTestIntegrationSmokeTest. AbstractTestIntegrationSmokeTest is in module presto-testing, and is connector-independent.

Thanks for the clarification.

So if I understand it correctly now we are running test for `SHOW CREATE TABLE` once for Parquet (via `testCreateTableForFormat`) and twice for ORC (via `testCreateTableForFormat` and `testShowCreateTable`).

Would code structure like this work:
```java 
    @Override
    @Test(enabled = false) 
    public void testShowCreateTable(){
        // superseded with #testShowCreateTable(FileFormat)
    }

    @Test(dataProvider = "allFormats")
    public void testShowCreateTable(FileFormat format) {
       ACTUAL TEST
    }
```

Reply:
That's a great idea!  Done.